index.html

```diff
@@ -4,3 +4,5 @@
   Dans l'idée maintenant je peux te proposer des modif' hyper facilement et tu
   peux choisir d'accepter ou pas :)
 </p>
+
+<p> Ok et je peux aussi faire des modifs, add, commit,push et PR </p>
```

Comment:
Le petit rond rouge la, c'est un "end of line". et ca veut dire qu'il manque une ligne vide en fin de fichier. C'est pas grave mais une ligne vide c'est ça permet au code de mieux être interpréter. Donc c'est un bon réflexe de les ajouter. Du coup je te proposition de changement directement via ce commentaire. Si tu l'acceptes ca devrait te rajouter directement la modif dans la branche. On peut tester

```suggestion
<p> Ok et je peux aussi faire des modifs, add, commit,push et PR </p>

```